wolox-training/rof-angular
# Implemented profile details view

## Summary

Implemented profile details view

## Test Plan

* Open the project
* Log into your account or create a new user and then log in, by clicking on buttons in header
* Click on profile button and confirm your personal details in the view.
* Logout
* Try to access /profile. You should be redirected to login.

## Trello Card 

 ttps://trello.com/c/gCq5XJ53

## Screenshot

![profile](https://cloud.githubusercontent.com/assets/2911971/10700784/6e846196-7996-11e5-99f6-2f230890a847.png)
